tests/cpptests/micro-benchmarks/benchmark_index_iterator.cpp

```diff
@@ -132,8 +132,6 @@ class BM_IndexIterator : public benchmark::Fixture {
         if (flags == Index_StoreNumeric) {
             FieldFilterContext fieldCtx = {.field = {false, 0}, .predicate = FIELD_EXPIRATION_DEFAULT};
             iterator = NewInvIndIterator_NumericQuery(index, &q_mock->sctx, &fieldCtx, nullptr, nullptr, -INFINITY, INFINITY);
```

Comment:
these Missing and Generic Queries are not relevant anymore for benchmarking?

Reply:
I guess since they only differ in the Revalidate logic, we assume they are not worth benchmarking? However I think this is assuming this will always remind like this. I think we should add them to keep it future proof?

Reply:
In practice, `Index_DocIdsOnly` cases use either the TagQuery or TermQuery APIs, so this is what we do in the benchmark as well now. Note that I didn't drop the `Index_DocIdsOnly` scenario, just the special treatment it got in the setup